dcc-portal-ui/app/scripts/genes/views/gene.html

```diff
@@ -196,144 +196,151 @@ <h1 data-ui-scrollfix=79 ><span class="t_badge t_badge__gene"></span>
+                        <span data-ng-bind-html="project.uiId | highlight: GeneCtrl.tableFilter.projects"></span>
+                    </a></td>
+                    <td><span data-ng-bind-html="project.uiPrimarySite | highlight: GeneCtrl.tableFilter.projects"></span></td>
+                    <td><span data-ng-bind-html="project.uiTumourType | highlight: GeneCtrl.tableFilter.projects"></span></td>
+                    <td><span data-ng-bind-html="project.uiTumourSubtype | highlight: GeneCtrl.tableFilter.projects"></span></td>
+                    <td class="text-right" style="position: relative; display: block">
 
                         <span data-ng-style="{width:project.uiAffectedDonorPercentage+'%'}"
-                          class="t_facets__facet__terms__active__term__bar"></span>
+                            class="t_facets__facet__terms__active__term__bar"></span>
                         <span style="position: relative">
-                        <a data-ng-href='/search?filters={{project.uiAdvQuery}}'
-                            data-ng-bind-html="project.uiFilteredDonorCount | highlight: GeneCtrl.tableFilter.projects">
-                        </a>
-                        /
-                        <a 
-                            data-ng-href='/search?filters={"donor":{"projectId":{"is":["{{ project.uiId }}"]},"availableDataTypes":{"is":["ssm"]}}}'
-                            data-ng-bind-html="project.uiSSMTestedDonorCount | highlight: GeneCtrl.tableFilter.projects">                        
-                        </a>
+                            <a data-ng-href='/search?filters={{project.uiAdvQuery}}'
+                                data-ng-bind-html="project.uiFilteredDonorCount | highlight: GeneCtrl.tableFilter.projects">
+                            </a>
+                            /
+                            <a data-ng-href='/search?filters={"donor":{"projectId":{"is":["{{ project.uiId }}"]},"availableDataTypes":{"is":["ssm"]}}}'
+                                data-ng-bind-html="project.uiSSMTestedDonorCount | highlight: GeneCtrl.tableFilter.projects">
+                            </a>
                         <em data-ng-bind-html="'('+ project.uiAffectedDonorPercentage + '%)' | highlight: GeneCtrl.tableFilter.projects"></em>
                         </span>
 
-                </td>
-                <td class="text-right">
-                    <span data-ng-if="project.uiMutationCount==null"><i class="icon-spinner icon-spin"></i></span>
-                    <span data-ng-if="project.uiMutationCount === 0">--</span>
-                    <a data-ng-if="project.uiMutationCount > '0'"
-                       data-ng-href='/search/m?filters={{project.uiAdvQuery}}'
-                       data-ng-bind-html="project.uiMutationCount | highlight: GeneCtrl.tableFilter.projects ">
-                    </a>
-                </td>
-            </tr>
-            </tbody>
-        </table>
+                    </td>
```

Comment:
These diffs are way too huge for me to say for sure
Personal preference would be to rather have incorrect indentation in a PR with functionality and minimize diff, and have a separate PR to add correct whitespace (and nothing but whitespace)